docs/module-types.md

```diff
@@ -115,3 +116,61 @@ where `wasm-module.wasm` is generated from:
   )
 )
 ```
+
+## HTML Modules
+
+[HTML Modules](https://github.com/w3c/webcomponents/blob/gh-pages/proposals/html-modules-explainer.md) are html files whose [HTMLDocument](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDocument) is (by default) exposed to the javascript code importing the html module. The
+html from the module **is not inserted into the DOM** unless you do so after importing it.
```

Comment:
Nit: consider saying “...not inserted into the DOM automatically, but the module importing it can do whatever it wants ( including inserting it into the DOM)”

Reply:
That’s what I was trying to say with “unless you do so after importing”

Sounds like that isn’t clear though, I can clarify it

Reply:
updated